devel/kokkos-devel/Portfile

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   cmake 1.1
+
+github.setup                kokkos kokkos 5b1a0e36a84994244f24efe3a199b324be455b05
+name                        kokkos-devel
```

Comment:
can this not be added as a subport of kokkos? I assume many parts of the Portfile will be identical

Reply:
I guess it can, if that is preferred. `kokkos` is not my port, so I went with a less intrusive option.

Reply:
@reneeotten Done.